scapy/asn1/asn1.py

```diff
@@ -32,6 +32,8 @@ def __init__(self, objlist=None):
         self.chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"  # noqa: E501
 
     def _fix(self, n=0):
+        # Because it's crypto, use SystemRandom (safer but ~30% slower)
```

Comment:
I think that we don't care =)

Reply:
Agreed. We need Scapy to be fast (or at least, less slow)!